packages/wrangler/pages/functions/filepath-routing.ts

```diff
@@ -52,7 +52,7 @@ export async function generateConfigFromFileTree({
             const declaration = node.declaration;
 
             // `export async function onRequest() {...}`
-            if (declaration.type === "FunctionDeclaration") {
+            if (declaration.type === "FunctionDeclaration" && declaration.id) {
```

Comment:
Functions that are defined as `export default function` do not have an `id`.
Should we instead add the `default` name to the `exportNames` list?

Reply:
No. Happy to exclude it like you've done.